src/tools/recvfuzz/CMakeLists.txt

```diff
@@ -0,0 +1,11 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+if(NOT QUIC_BUILD_SHARED)
+    add_compile_definitions(QUIC_BUILD_STATIC)
+endif()
+add_quic_tool(recvfuzz recvfuzz.cpp)
+target_include_directories(recvfuzz PRIVATE ${PROJECT_SOURCE_DIR}/src/core)
+if (BUILD_SHARED_LIBS)
+    target_link_libraries(recvfuzz core)
```

Comment:
Seeing this build error in CI:
```
core.lib(library.obj) : error LNK2005: MsQuicOpenVersion already defined in msquic.lib(msquic.dll) [D:\a\msquic\msquic\build\windows\x64_schannel\src\tools\recvfuzz\recvfuzz.vcxproj]
```

Reply:
as discussed, the build has to be with the -static flag. There is another workaround where I can include msquic in the target_link_libraries